src/components/text-editor/__internal__/editor-link/editor-link.component.js

```diff
@@ -29,12 +29,11 @@ const EditorLink = ({ children, contentState, entityKey, ...rest }) => {
 
   return (
     <StyledLink
-      href={validUrl}
+      href={!editMode ? validUrl : undefined}
       title={validUrl}
-      aria-label={validUrl}
+      aria-label={!editMode ? validUrl : undefined}
```

Comment:
Seeing as `href` and `aria-label` have the same ternary statement, could this not be assigned to `const` instead.

Reply:
I think that it's easier to read when it's presented like that as you instantly know why it's not rendered here

Reply:
That's fine. Personal preference at the end of the day. Happy to approve the PR still 😄